scaleapi/__init__.py

```diff
@@ -782,20 +782,29 @@ def create_project(
         params: Dict = None,
         rapid: bool = False,
         studio: bool = False,
+        dataset_id: Optional[str] = None,
     ) -> Project:
         """Creates a new project.
         https://docs.scale.com/reference#project-creation
 
         Args:
             project_name (str):
                 Project name
-
             task_type (TaskType):
                 Task Type i.e. `TaskType.ImageAnnotation`
-
             params (Dict):
-                Project parameters to be specificed.
+                Project parameters to be specified.
                 i.e. `{'instruction':'Please label the kittens'}`
+            rapid (bool):
```

Comment:
Two boolean variables that are only valid when xor'd. Great 🙈

Kudos on the doc string though!